File: Beginner/fancyQuote/fancyQuote.c
#include <stdio.h>
#include <assert.h>
#include <string.h>

int main(void){
	int T,flag;
	char S[101];

	scanf("%d",&T);
	assert(T>=1 && T<=50);

	while(T>0){
		scanf(" %[^\n]",S);

		for(int i=0;i<strlen(S);i++){
			assert((S[i]>=97 && S[i]<=122) || S[i]==' ');
		}

		flag = 0;

		for(int i=0;S[i]!='\0';i++){
			if((S[i-1]==' ' || i==0) && S[i]=='n' && S[i+1]=='o' && S[i+2]=='t' && (S[i+3]==' ' || S[i+3]=='\0')){
				flag = 1;
				break;
			}
		}

		if(flag==1){
			printf("Real Fancy\n");
		}else{
			printf("regularly fancy\n");
		}

		T--;
	}

	return 0;
}

File: Beginner/fancyQuote/fancyQuote.java
//run as java -ea fancyQuote
import java.util.Scanner;

public class fancyQuote{
	public static void main(String[] args){
		int T,flag;
		String S;
		Scanner scan = new Scanner(System.in);

		T = scan.nextInt();
		scan.nextLine();
		assert(T>=1 && T<=50) : "T must be 1 <= T <= 50";

		while(T>0){
			S = scan.nextLine();
			assert(S.length()>=1 && S.length()<=100) : "String S must be of length 1 <= S <= 100";

			for(int i=0;i<S.length();i++){
				assert((S.charAt(i)>=97 && S.charAt(i)<=122) || S.charAt(i)==' ') : "String S must be made up of either lower case English letters or a space";
			}

			String[] A = S.split(" ");

			flag = 0;

			for(int i=0;i<A.length;i++){
				if(A[i].equals("not")){
					flag = 1;
				}
			}

			if(flag==1){
				System.out.println("Real Fancy");
			}else{
				System.out.println("regularly fancy");
			}

			T--;
		}
	}
}